packages/react/src/component-tests/IcClassificationBanner/IcClassificationBanner.cy.tsx

```diff
@@ -0,0 +1,110 @@
+/* eslint-disable react/jsx-no-bind */
+/// <reference types="Cypress" />
+
+import React from "react";
+import { IcClassificationBanner } from "../../components";
+import { CYPRESS_AXE_OPTIONS } from "../../../cypress/utils/a11y";
+import { mount } from "cypress/react";
+
+const DEFAULT_TEST_THRESHOLD = 0.2;
```

Comment:
This is a 20% threshold, which I think is maybe overkill? Did you try it out at lower percentages?